storage/queue.go

```diff
@@ -114,7 +114,7 @@ type queueLog struct {
 
 func (l queueLog) Infof(logv bool, format string, a ...interface{}) {
 	if logv {
-		log.Infof(l.prefix+format, a...)
+		log.InfoDepth(1, l.prefix+format, a...)
```

Comment:
According to Ben, this is not equivalent. You'd need to add an `InfofDepth` method.